bin/pycbc_live

```diff
@@ -375,6 +377,7 @@ class LiveEventManager(object):
         event = SingleCoincForGraceDB(
             live_ifos,
             coinc_results,
+            padata=padata,
```

Comment:
Should this be `self.padata`?

Reply:
ah, this only works because 'padata' has already been assigned a value .. so yes, probably should be fixed